aspnet/web-pages/content-guide.md

```diff
@@ -45,7 +45,7 @@ WebMatrix is a set of tools that provides an easy way to build Web sites for Mic
 
 **Read**
 
-- [Microsoft Web Camps training kit for WebMatrix](http://trainingkit.webcamps.ms/WebMatrix.htm)
+
```

Comment:
I think you can substitute this link instead of removing the old one: https://github.com/Microsoft-Web/WebCampTrainingKit